src/charts/bullet.js

```diff
@@ -309,7 +306,7 @@ export default function module() {
                 .enter()
                 .append('line')
                 .attr('class', 'marker-line')
-                .attr('stroke', measureColor)
+                .attr('stroke', markerColor)
```

Comment:
Are we sure we need to use marketColot instead? Just want to make sure that nothing looks off when we switch colors in demo

Reply:
Yep, that way we can configure that color too. I think it was intended, but we missed it somehow.